app/common/state/tabState.js

```diff
@@ -183,6 +183,7 @@ const api = {
   getPersistentState: (state) => {
     // TODO(bridiver) - handle restoring tabs
     state = makeImmutable(state)
+    // TODO(bsclifton) - if above TODO is implemented, we may not want to save/restore all tab data (ex: alerts, etc)
```

Comment:
we definitely don't want to restore alerts because they won't be connected to the original method call anyway